crates/turbo-tasks-fetch/src/lib.rs

```diff
@@ -2,12 +2,21 @@
 
 use anyhow::Result;
 use turbo_tasks::primitives::{OptionStringVc, StringVc};
+use turbo_tasks_fs::FileSystemPathVc;
+use turbopack_core::issue::{Issue, IssueSeverityVc, IssueVc};
 
 pub fn register() {
     turbo_tasks::register();
+    turbopack_core::register();
     include!(concat!(env!("OUT_DIR"), "/register.rs"));
 }
 
+#[turbo_tasks::value]
+pub enum FetchResult {
+    Ok(HttpResponseVc),
+    Err(FetchErrorVc),
+}
```

Comment:
Couldn't this also be a 
```suggestion
#[turbo_tasks::value(transparent)]
pub struct FetchResult(Result<HttpResponseVc, FetchErrorVc>);
```